### PR TITLE
Fixes #7 update exception view

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/OSPSuite.Core/IApplicationConfiguration.cs [hunk 1]
```diff
@@ -71,5 +70,10 @@ public interface IApplicationConfiguration
       /// Returns a possible enumeration containg the path of user settings that can be loaded. (Starting from the most recent one down to the first available one)
       /// </summary>
       IEnumerable<string> UserApplicationSettingsFilePaths { get; }
+
+      /// <summary>
+      /// Returns the url of the issue tracker
+      /// </summary>
+      string IssueTrackerUrl { get; }
```
Comment (msevestre) [inline src/OSPSuite.Core/IApplicationConfiguration.cs]: This needs to be defined in all applications

File: src/OSPSuite.Core/Domain/Constants.cs [hunk 2]
```diff
@@ -62,11 +62,11 @@ public static class Constants
       public const string NAN = "<NaN>";
       public static readonly string CHILD = "Child";
 
-      public static readonly string SUPPORT_EMAIL = "support@systems-biology.com";
       public static readonly string PRODUCT_SITE = "www.open-systems-pharmacology.org";
       public static readonly string PRODUCT_SITE_DOWNLOAD = "http://setup.open-systems-pharmacology.org";
       public static readonly string HELP_NAMESPACE = "Open Systems Pharmacology Suite.chm";
       public static readonly string LICENSE_AGREEMENT_FILE_NAME = "Open Systems Pharmacology Suite License.pdf";
+      public static readonly string FORUM_SITE = "forum.open-systems-pharmacology.org";
```
Comment (msevestre) [inline src/OSPSuite.Core/Domain/Constants.cs]: Remove support email and add forum

File: src/OSPSuite.Assets/UIConstants.cs [hunk 3]
```diff
@@ -236,17 +241,19 @@ public static string DoYouWantToDeleteDirectory(string newDirectoryName)
...
-         sb.AppendLine($"To best address the error, please send us an email at <B>{supportEmail}</B> with:");
-         sb.AppendLine("    1 - the information below");
-         sb.AppendLine("    2 - the steps you took prior to the problem emerging");
-         sb.AppendLine("    3 - if possible, attach your project file to the email (do not send confidential information by email).");
+         sb.AppendLine("To best address the error, please enter an issue in our issue tracker:");
+         sb.AppendLine($"    1 - Visit <b>{issueTrackerUrl}</b> or click on the link below");
+         sb.AppendLine("    2 - Click on the <b>New Issue</b> button");
+         sb.AppendLine($"    3 - Copy the information below by using the <b>{CopyToClipboard}</b> button and paste it in the issue description");
+         sb.AppendLine("    2 - Describe the steps you took prior to the problem emerging");
+         sb.AppendLine("    3 - if possible, attach your project file to the issue (do not attach confidential information).");
```
Comment (Yuri05) [inline src/OSPSuite.Assets/UIConstants.cs]: 1 2 3 2 3 or 1 2 3 4 5? ;-)
Also I would put "_Describe the steps..._" before "_Copy the information..._"

Comment (msevestre) [inline src/OSPSuite.Assets/UIConstants.cs]: lol yeah. I was just testing to make sure you were reviewing everything :)

Comment (msevestre) [inline src/OSPSuite.Assets/UIConstants.cs]: done

**Comment by msevestre** [comment]
Going to merge this one Juri so that I can push PKSim and MoBi

**Pull request merged**